tensorflow/python/debug/BUILD

```diff
@@ -727,6 +727,7 @@ sh_test(
     name = "examples_test",
     size = "medium",
     srcs = ["examples/examples_test.sh"],
+    tags = ["manual"],
```

Comment:
I think the flaky timeout should have been fixed by CL/156945796, which is already pushed out to GitHub and increases the test size to medium to prevent flaky timeouts. So this change is not necessary.